cg/apps/crunchy/files.py

```diff
@@ -11,6 +12,30 @@
 LOG = logging.getLogger(__name__)
 
 
+# Methods to get file information
+def get_log_dir(file_path: Path) -> Path:
+    """Return the path to where logs should be stored"""
+    return file_path.parent
+
+
+def get_sbatch_path(log_dir: Path, compression: str, run_name: str = None) -> Path:
```

Comment:
This can be simplified I think! But making 2 methods - one returns decompress name and one compress name... then there wont be need to pass compression="fastq"... But its like, just a picky opninion

Reply:
Splitting into two!